help_to_heat/frontdoor/eligibility.py

```diff
@@ -29,6 +29,11 @@ def calculate_eligibility(session_data):
     council_tax_band = session_data.get("council_tax_band")
     country = session_data.get("country")
     benefits = session_data.get("benefits")
+    property_type = session_data.get("property_type")
+
+    # "Scenario 0"
```

Comment:
Presuming all park homes are automatically eligible for GBIS

Reply:
Presuming? Or is that the intended/specified behaviour?

Reply:
Looking at the powerpoint, they didn't specify what schemes park homes are eligible for, but by the look of the flow, park homes will automatically be eligible (they don't have EPC etc...). @LadunOmideyiSoftwire  please shout if you think we need to confirm with the client etc.!